be/src/vec/sink/vtablet_sink.cpp

```diff
@@ -605,6 +605,21 @@ Status VNodeChannel::add_block(vectorized::Block* block, const Payload* payload,
 
     SCOPED_RAW_TIMER(&_stat.append_node_channel_ns);
     if (is_append) {
+        if (_cur_mutable_block && !_cur_mutable_block->empty()) {
```

Comment:
Please add a comment and add a regression test.

Reply:
done